### PR TITLE
feat(DataGrid): Implement Home/End navigation with Control

### PR DESCRIPTION
**Pull request by ling1726**
CTRL+Home will focus the first focusable element in the first row.

CTRL+End will fous the last focusable elemtn in the last row.

Also adds cypress tests to validate the focusing behaviour.

### CODE REVIEW

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/30ad74d01e550256a7f62dd8760d393d54473959'>30ad74d01e550256a7f62dd8760d393d54473959</a></sub>
<div id="bundleSizeComment9423"/>

**Comment by fabricteam** [comment]
## [Perf Analysis (`@fluentui/react-components`)](https://github.com/microsoft/fluentui/wiki/Perf-Testing)
<p>No significant results to display.</p><details><summary>All results</summary><p>
  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>Avatar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26964/perf-test-react-components/Avatar-mount_base.html">1218</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26964/perf-test-react-components/Avatar-mount.html">1297</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Button</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26964/perf-test-react-components/Button-mount_base.html">873</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26964/perf-test-react-components/Button-mount.html">947</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Field</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26964/perf-test-react-components/Field-mount_base.html">1991</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26964/perf-test-react-components/Field-mount.html">1950</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProvider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26964/perf-test-react-components/FluentProvider-mount_base.html">1565</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26964/perf-test-react-components/FluentProvider-mount.html">1530</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26964/perf-test-react-components/FluentProviderWithTheme-mount_base.html">590</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26964/perf-test-react-components/FluentProviderWithTheme-mount.html">637</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26964/perf-test-react-components/FluentProviderWithTheme-virtual-rerender_base.html">527</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26964/perf-test-react-components/FluentProviderWithTheme-virtual-rerender.html">583</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26964/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount_base.html">628</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26964/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount.html">629</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>InfoButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26964/perf-test-react-components/InfoButton-mount_base.html">571</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26964/perf-test-react-components/InfoButton-mount.html">497</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MakeStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26964/perf-test-react-components/MakeStyles-mount_base.html">1898</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26964/perf-test-react-components/MakeStyles-mount.html">1869</a></td>
            <td>50000</td>
            <td></td>
           </tr>
<tr>
            <td>Persona</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26964/perf-test-react-components/Persona-mount_base.html">3040</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26964/perf-test-react-components/Persona-mount.html">2828</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SpinButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26964/perf-test-react-components/SpinButton-mount_base.html">2449</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26964/perf-test-react-components/SpinButton-mount.html">2410</a></td>
            <td>5000</td>
            <td></td>
           </tr></table></p></details>

<div id="perfComment9425"/>

**Comment by fabricteam** [comment]
## 🕵 fluentuiv9 No visual regressions between this PR and main
<div id="vrtCommentfluentuiv9"/>

**Review by Hotell — APPROVED** [review]
all good for v-build

File: change/@fluentui-react-table-7bd6b73b-5074-4444-95cb-c04f2a5e75a5.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
+  "comment": "feat(DataGrid): Implement Home/End navigation with Control",
```
Comment (Hotell) [inline change/@fluentui-react-table-7bd6b73b-5074-4444-95cb-c04f2a5e75a5.json:3]:
```suggestion
  "comment": "feat: Implement Home/End navigation with Control",
```

**Review by george-cz — APPROVED** [review]

**Pull request merged**